src/content/overview/observability/observability-platform-api/_index.md

```diff
@@ -0,0 +1,111 @@
+* [Global overview](#global-overview)
+  * [How it works](#how-it-works)
+    * [Read path](#read-path)
+    * [Write path](#write-path)
+* [Adding datasources](#adding-datasources)
+* [Limitations](#limitations)
+
+## What it is
+
+The observability platform already ingests and allows to explore system and application observability data from inside Giant Swarm managed clusters by default.
+
+The `observability platform API` opens up the observability platform to be used from the outside - which means any resource not managed by Giant Swarm. You can ingest observability data from any source by sending them to the `observability platform API`. Additionally the API allows to query observability data from wherever you want. 
+
+This allows you to for example ingest observability data from a SaaS-Database that might be a dependency of one of your workloads on a Giant Swarm managed cluster. Or you could explore the observability platforms data from your own, remote Grafana instance or any other observability tooling.
+
+**Note:** At this point the `observability platform API` only allows the ingestion of logs and events. The ingestion of metrics will follow in a later release. Keep an eye on our [changes and releases]({{< relref "/changes/observability-platform/" >}}) or this document for updates.
+
+The `observability platform API`s main objectives are to:
+
+* provide a secure access to our observability platform from outside of Giant Swarm managed clusters.
+* enable you to ingest and access observability data from anywhere.
+* sanitize (sampling, relabelling, ...) observability data to align with our general data standards in the observability platform.
+* support for the OpenTelemetry Protocol (OTLP).
+
+## How it works
+
+### Global overview
+
+Here is a graph showing the general architecture for the API :
+
+![api architecture](./observability-platform-api-graph.png)
+
+### How it works
+
+Let's explore the 2 main workflows from the customer's perspective.
+
+#### Read path
+
+Let's consider the following situation : a customer wants to access Giant Swarm observability data from his own Grafana running on his premises.
+
+To do so, the customer will first log into the Grafana instance using the OIDC provider hosted on his premises. Once authenticated, he will perform a query from the "Explore" panel or simply use dashboards which in any case will trigger a request sent to the Observability Platform API url.
+
+There, depending on the path the requests is trying to access (i.e either trying to access logs or metrics) it will go through one of 2 ingresses sharing the same host : one in the loki namespace and the other in the mimir one. Note that we decided to have 2 ingresses sharing the same host instead of having 1 ingress handling all paths because usually, ingresses only forward requests in the same namespace they are deployed in. There are ways to go around this, but those are a bit tricky and definitely not straight forward.
```

Comment:
```suggestion
There, depending on the path the request is trying to access (i.e. either trying to access logs or metrics), it will go through one of two ingresses sharing the same host: one in the Loki namespace and the other in the Mimir one. Note that we decided to have two ingresses sharing the same host instead of having one ingress handling all paths because, usually, ingresses only forward requests in the same namespace in which they are deployed. There are ways to go around this, but those are a bit tricky and not straightforward.
```